main/file.c

```diff
@@ -138,18 +142,94 @@ static sub_status_t sleep_unlock(void){
 
 esp_now_send_status_t sub_get_last_message_status(void){
 
-    return ESP_NOW_SEND_SUCCESS;
+    return sub_struct.last_message_status;
 }
 
 sub_status_t sub_enable_sleep(void){
     return true;
 }
 
-static sub_status_t send_cb(){
+static void send_cb(const uint8_t *mac, esp_now_send_status_t status){
 
-    return SUB_OK;
+    sub_send_cb_data_t send_data;
+
+    send_data.message_status = status;
+
+    memcpy(send_data.mac, mac, ESP_NOW_ETH_ALEN);
+
+    xQueueSend(sub_struct.send_cb_queue, &send_data, 0)
```

Comment:
Nie mogę znaleźć miejsca gdzie tworzysz kolejkę send_cb_queue i receive_cb_queue. Jak jest gdzieś to podlinkuj w odpowiedzi. Jak coś to najlepiej żeby obie miały rozmiar 1.